apps/accelerate/chatllama/chatllama/rlhf/reward.py

```diff
@@ -167,7 +175,11 @@ def load(self, path: Optional[str] = None) -> None:
             path (str): path to the model
         """
         if path is None:
-            path = self.config.model_folder + "/" + self.config.model + ".pt"
+            if self.config.model in hf_models:
+                model_name = os.path.split(self.config.model)[-1]
+            else:
+                model_name = self.config.model
+            path = self.config.model_folder + "/" + model_name + ".pt"
```

Comment:
please use `os.path.join`
```suggestion
            path = os.path.join(self.config.model_folder, f"{model_name}.pt")
```